<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="EPINHYDW0C77" failures="1" tests="7" name="Test" time="63.317" errors="0" timestamp="2021-11-23T18:32:00 IST">
  <testcase classname="testCases.TC001_HomePageValidation" name="verifySizesFilter" time="31.564"/>
  <testcase classname="testCases.TC001_HomePageValidation" name="verifyOrderBySorting" time="4.576"/>
  <testcase classname="testCases.TC001_HomePageValidation" name="addItemsInCart" time="2.477"/>
  <testcase classname="testCases.TC002_CartValidation" name="cartFunctionality" time="2.493"/>
  <testcase classname="testCases.TC002_CartValidation" name="updateItemsInCart" time="6.179"/>
  <testcase classname="testCases.TC002_CartValidation" name="numberOfItemsInCartAfterBrowserRefresh" time="3.484">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at testCases.TC002_CartValidation.numberOfItemsInCartAfterBrowserRefresh(TC002_CartValidation.java:111)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- numberOfItemsInCartAfterBrowserRefresh -->
  <testcase classname="testCases.TC002_CartValidation" name="checkout" time="0.079"/>
</testsuite> <!-- Test -->
